### DISCUSSION TITLE
INPUT ERROR: Element type not properly defined

### DISCUSSION THREAD

**Discussion by PetitPipi** [Anura3D/Anura3D_OpenSource#163] [Q&A] [answered]
Hi,
I encountered this error when I tried to generate A3D files.  The model was a 3D model involving a cylinder.  
Any idea how to cope with this? 
The message was as follows:
![Screenshot 2023-11-27 143546](https://github.com/Anura3D/Anura3D_OpenSource/assets/106621276/8dacc511-e2a2-4f92-8adf-c2104111e637)

Regards!

**Comment by PetitPipi** [comment]
Update:
I think this issue is linked to meshing.   I bumped into the same problem with another 3D model. This time I used a semi-structured mesh similar to the one used in the consolidation example instead of an unstructured mesh, and then this error disappeared.

**Reply by murphy-john** [reply]
Which version of GiD are you using? There seems to be an issue when users use the GiD developers versions 16.X.Xd instead of the release version 16.X.X.

**Reply by PetitPipi** [reply]
I used the release version 16.0.6

**Reply by pmarveggio** [reply, marked as answer]
If you are running a 3D model, just be sure to set Mesh->Quadratic type->Quadratic and then it should run
